src/App.tsx

```diff
@@ -1,26 +1,188 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useRef, useState } from 'react';
 
-const USER_ID = 0;
+import { UserWarning } from './UserWarning';
+import { USER_ID, getTodos, postTodos, deleteTodos } from './api/todos';
+import { TodoList } from './Components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import { Footer } from './Components/Footer/Footer';
+import { StatesOfFilter } from './types/StatesOfFilter';
+import { getFilteredTodos } from './utils/getFilteredTodos';
+import { ErrorNotification } from './Components/ErrorNotification';
+import { getRandomNumberId } from './utils/getRandomNumberId';
+import { getCountOfCompletedTodos } from './utils/getCountOfCompletedTodos';
 
 export const App: React.FC = () => {
+  const [newTitle, setNewTitle] = useState<string>('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<StatesOfFilter>(StatesOfFilter.All);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isAllDeleting, setIsAllDeleting] = useState<boolean>(false);
```

Comment:
Better to store array of todos' `id`s that are in processing and should be with loader

Reply:
I implemented it in the third task. Can I skip it in this task?

Reply:
> I implemented it in the third task. Can I skip it in this task?

Ok